### PR TITLE
CORE-17055 Add CI job for plugin smoke tests

### PR DESCRIPTION
**Pull request by YashNabar**
Adds a CI job for running Corda CLI plugin smoke tests against the Combined Worker. This job is currently set up to be triggered manually during development, and is not run on a schedule or as a PR-gate.
https://ci02.dev.r3.com/job/Corda5/job/corda-runtime-os-combined-worker-plugins-smoke-tests/

Also includes a fix for saving the last created Group ID in the `onboard-mgm` command.

### CODE REVIEW

File: .ci/JenkinsfileCombinedWorkerPluginsSmokeTests [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                    findBuildScans()
+                    splunkLogGenerator()
+                    pipelineUtils.getPodLogs("postgres")
+                    pipelineUtils.dropPostgresDB(postgresCredentialsId, env.CORDA_DEV_CLUSTER_DB_NAME)
+            }
+            archiveArtifacts artifacts: 'forward.txt, workerLogs.txt, podLogs.txt', allowEmptyArchive: true
+            sh 'rm -f forward.txt workerLogs.txt podLogs.txt'
+        }
+        failure {
+            sendSlackNotifications("danger", "BUILD FAILURE - Combined Worker E2E Tests", true, "#corda-corda5-build-notifications")
```
Comment (mcgovc) [inline .ci/JenkinsfileCombinedWorkerPluginsSmokeTests]: Not sure if you want to send a Slack notification at all, but if keeping this I would say it should be updated to reflect the correct tests. 
```suggestion
            sendSlackNotifications("danger", "BUILD FAILURE - Combined Worker Plugins Smoke Tests", true, "#corda-corda5-build-notifications")
```

Comment (YashNabar) [reply .ci/JenkinsfileCombinedWorkerPluginsSmokeTests]: Thanks, I've removed that in [2370f04](https://github.com/corda/corda-runtime-os/pull/4707/commits/2370f0451b6cc6bebb2d791ac35c045320c9c042)

**Comment by YashNabar** [comment]
Plugin smoke tests: https://ci02.dev.r3.com/blue/organizations/jenkins/Corda5%2Fcorda-runtime-os-combined-worker-plugins-smoke-tests/detail/PR-4707/4/tests

**Review by dimosr — DISMISSED** [review]
LGTM - I'll leave the BLT team to review the Jenkins/gradle files in a bit more detail as they are the experts on that area.

File: .ci/JenkinsfileCombinedWorkerPluginsSmokeTests [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                        junit allowEmptyResults: true, testResults: '**/test-results/**/TEST-*.xml'
+                    }
+                }
+        }
+    }
+    post {
+        always {
+            script {      
+                    findBuildScans()
+                    splunkLogGenerator()
```
Comment (ronanbrowne) [inline .ci/JenkinsfileCombinedWorkerPluginsSmokeTests]:
```suggestion
```

Minor,  You can remove this, as Splunk will be discontinued at the end of this month.

**Review by ronanbrowne — CHANGES_REQUESTED** [review]
Minor comment

**Review by mcgovc — APPROVED** [review]
LGTM

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Review by dimosr — APPROVED** [review]

**Pull request merged**